front/src/modules/game/table/deck/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { pullCard } from '../../../../state/game';
+import './style.css';
+
+class Deck extends PureComponent {
+
+  render() {
+    const { onPullCard } = this.props;
+    return (
+      <div >
+        <div className="deck" color="Pull card" onClick={() => onPullCard()}>
+          Pull card
+        </div>
+      </div>
+    );
+  }
+};
+
+const mapStateToProps = (state) => ({
+
+});
+
+const mapDispatchToProps = (dispatch) => ({
```

Comment:
`mapDispatchToProps` can be an object, this will reduce callbacks creation.

`mapDispatchToProps = {
  onPullCard: pullCard
}`

Reply:
i've improved here and in some others files